src/ext/Firewall/wixext/FirewallDecompiler.cs

```diff
@@ -499,4 +501,14 @@ private void FinalizeFirewallExceptionTable(TableIndexedCollection tables)
             }
         }
     }
+
+    internal static class StringExtensions
+    {
+        public static string ToCamelCase(this string str)
+        {
+            var camelCase = str[0].ToString().ToLowerInvariant() + str.Substring(1);
```

Comment:
Should this protect against `null` and `String.Empty`?

Reply:
I'd be more worried about something public, because surely I will never make such a mistake...